hydra.py

```diff
@@ -40,6 +40,9 @@ def _hydra(self, fn, endpoint, parameters=None, payload=None):
         return response.json()
 
     def get_account_notes(self, account):
+        # Adding this additional check to force the (seemingly) random chance of the account showing up as a float (producing the extra .0) into the correct format
+        if isinstance(account, float):
+            account = str(int(account))
```

Comment:
We probably don't need the wrapping `str`.  And we probably do want this for our other Hydra methods, which also format the passed-in account.  I'm guessing we should put the `int` cast in `telemetry.py`'s `ebs_account` function, like:

```python
return int(subscription['ebs_account'])
```

If that makes sense and you reroll, can you rebase on master to avoid the merge you have today:

```console
$ git --no-pager log --oneline --graph -6 origin/pr/42
* 278ad8c (origin/pr/42) adding a check and type force to prevent the .0 issue we saw previously
*   5b9e17e Merge pull request #1 from openshift/master
|\  
| * c220f69 (HEAD -> master, wking/master, origin/pr/40, origin/master, origin/HEAD) hydra: ISO-8601 / RFC-3339 format for expiryDate
| * 0b2577f (origin/pr/39) hydra: use 365 days as a substitute for years
| * 5178482 (origin/pr/38) hydra: expiryDate is now required to create a note
|/  
*   92e4e93 Merge pull request #35 from vrutkovs/blockit
|\  
```

Reply:
You are right, I thought since we were dropping it into a string, we'd need it to ALSO be a string, but I just checked on a couple versions of python and since we are using `format`, it can still stay as the int, not the string.
Also yeah, that Does make sense to me, moving it into the place we introduce the account number.

For clarity's sake, should I kill this PR before the rebase, then?

Reply:
> For clarity's sake, should I kill this PR before the rebase, then?

You can force-push whatever to your existing branch, and it will update this PR.  So no need to move to a new PR, but we can if that's easier for you.